services/siteApi.js

```diff
@@ -26,6 +26,18 @@ exports.fetchAll = () => {
   });
 }
 
+exports.fetchUsersOfSite = (siteId) => {
+  return rp({
```

Comment:
Be aware that request-promise is deprecated and should be migrated to node-fetch. This has apparently already been done in `providers/oauthProvider.js` above.

You might want to consider to migrate this file while your at it.